tests/test_cli_main.py

```
@@ -123,3 +123,55 @@ def _make_manager(config_dir):
+
+        code = (
+            "import sys\n"
+            "from cli.main import app\n"
+            "heavy_modules = ['sqlalchemy', 'pydantic', 'watchdog']\n"
+            "loaded = [m for m in heavy_modules if m in sys.modules]\n"
+            "if loaded:\n"
+            "    print(','.join(loaded))\n"
+            "    sys.exit(1)\n"
+            "sys.exit(0)\n"
+        )
+        result = subprocess.run([sys.executable, "-c", code], capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)